pkg/cli/webhook/webhook.go

```diff
@@ -120,17 +121,17 @@ func askProvider() (Interface, error) {
 	}
 
 	if answer == "GitHub" {
-		return &gitHubConfig{}, nil
+		return &gitHubConfig{Hosted: true}, nil
```

Comment:
is it uncondinially true to github enteprise?

Reply:
if we don't find https://github.com in url
we ask user which provider do you want to configure
so that would be ghe

Reply:
ah okay nice, feel free to add a comment about those things next time!